parlai/tasks/convai2/worlds.py

```diff
@@ -44,8 +43,8 @@ def load_personas(opt):
                     )
                 if t.startswith('your persona:'):
                     a2_persona += t + '\n'
-            personas.add(a1_persona)
-            personas.add(a2_persona)
+            personas.add(a1_persona.rstrip('\n'))
```

Comment:
why do we strip the new line here? is it becuase there is an extra newline at the end? maybe we shouldj ust edit the way we grab the personas. something like 
```
a1_persona = []
...
    a1_persona.append(t.replace("partner's persona:", "your persona:")
...
personas.add('\n'.join(a1_persona)

```

Reply:
sure, that would do the same thing

Reply:
(but yes, the extra newline was a bug)